packages/helix/project.bri

```diff
@@ -0,0 +1,48 @@
+import * as std from "std";
+import git from "git";
+import rust, { cargoBuild } from "rust";
+
+export const project = {
+  name: "helix",
+  version: "25.01.1",
+  repository: "https://github.com/helix-editor/helix.git",
+};
+
+const source = Brioche.gitCheckout({
+  repository: project.repository,
+  ref: project.version,
+});
+
+export default function helix(): std.Recipe<std.Directory> {
+  return cargoBuild({
+    env: {
+      HELIX_DISABLE_AUTO_GRAMMAR_BUILD: "1",
+    },
+    path: "helix-term",
+    runnable: "hx",
+    source,
+  });
+}
+
+export async function test(): Promise<std.Recipe<std.File>> {
+  const script = std.runBash`
+    helix --version | tee "$BRIOCHE_OUTPUT"
+  `
+    .dependencies(rust, helix)
```

Comment:
Once the package built, to test it, you don't need the `rust` package:

```suggestion
    .dependencies(helix)
```

Thus, after that, you could simply remove `rust` from the imports